docker-entrypoint.sh

```diff
@@ -128,14 +128,15 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 		sed -ri -e 's/\r$//' wp-config*
 
 		if [ ! -e wp-config.php ]; then
-			awk '/^\/\*.*stop editing.*\*\/$/ && c == 0 { c = 1; system("cat") } { print }' wp-config-sample.php > wp-config.php <<'EOPHP'
+			awk '/^\/\*.*stop editing.*\*\/$/ && c == 0 { c = 1; system("cat") } { print }' wp-config-sample.php > wp-config.php <<'EOPHP' <<EOBASH
```

Comment:
This multi-heredoc syntax seemed too good to be true, so I tested and it does appear to be too good to be true: :disappointed:
```console
$ cat <<EOF1 <<EOF2
> hi
> EOF1
> hi2u
> EOF2
hi2u
```

It appears the second overwrites the first.  However, all three implementations of `awk` that I tested (busybox, `mawk`, and `gawk`) include an `ENVIRON` variable which we can use.  Here's my suggested modification:

```bash
awk '
	/^\/\*.*stop editing.*\*\/$/ && c == 0 {
		c = 1
		system("cat")
		if (ENVIRON["WORDPRESS_ADDITIONAL_CONFIG"]) {
			print ENVIRON["WORDPRESS_ADDITIONAL_CONFIG"]
		}
	}
	{ print }
' wp-config-sample.php > wp-config.php <<'EOPHP'
...
EOPHP
```

Reply:
Strange, the actual implementation works for me

Reply:
Im using different heredoc styles, maybe that’s why 🤔